### ISSUE TITLE
Searchable list answer not persisting from the review page in preview/test

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#3970]
The user notes that the answers they added and then went back to change from the review page did not persist in the form, although they did update and persist properly on the review page after editing.

> At the review section, when a user clicks on 'go back', could their answers stay visible? When I clicked on 'go back', the answers I filled out appeared to have been removed but when I clicked on continue to take me back to the review section, my previous answers were still there. Anything new I added into the blank fields was changed but any fields I had left blank were populated with my original/previous responses.

### Steps to Recreate:

1. Create a form with multiple pages
2. Add a Searchable list element to one of the pages
3. (This happens for Draft AND Published forms) Fill out the form (including the searchable list) and then click "Go Back" when you hit the review page. Go back to the page with the searchable list
4. See error: the input is blank (but if you continue back to the review page, the answer still shows up)

**Comment by connorscarolyns** [comment]
@timarney @Abi-Nada  Follow-up feedback from the client:
Hi Carolyn,
 
I tried it again and this time the fields showed me my previous responses. Might have just been an odd one-time glitch then. There’s one field that I noticed did continue to stay blank for me; not sure if that’s just the nature of that field type though.  
 
I filled out parts of my test form, including the OHEG section indicated below (field type = searchable list). If from this screen I click on ‘go back’ the screen I get taken to is the last page, which is the OHEG section. The next screenshot below shows the field is blank. Same if I click on ‘edit’ for this specific section.

![image](https://github.com/cds-snc/platform-forms-client/assets/53620052/41ab3e3e-049f-4877-bd5e-0d80c19d101c)
 
If I leave this field blank and click ‘continue’ to return to the review page, my original selection is still there. If I change it, the change is applied after I click on ‘continue’.
 
![image](https://github.com/cds-snc/platform-forms-client/assets/53620052/9f36d14a-8043-4245-b85f-91d1dccdec92)
